README.md

```diff
@@ -148,6 +146,9 @@ npx vercel --prod
 ## Next steps
 
 - [Join the Sanity community](https://slack.sanity.io/)
```

Comment:
I would suggest these links should be more actionable. It's not clear why someone should join the slack community. If they're new they almost certainly don't know what a schema and GROQ is. 

Try to make it actionable, easier to understand and helpful, e.g.
"Join our Slack community to ask questions and get help"
"How to edit my content structure?"
"How to query content?"

Reply:
Great points 👍